demos/src/full-bleed-image-left.mustache

```diff
@@ -0,0 +1,22 @@
+<div class="o-topper o-topper--full-bleed-image-left">
+	<div class="o-topper__content">
+		<div class="o-topper__tags">
+			<a href="https://www.ft.com/life-arts" class="o-topper__topic">Life &amp; Arts</a>
+		</div>
+		<h1 class="o-topper__headline o-topper__headline--large" data-trackable="header">
+			<span class="article-classifier__gap">What makes a great surgeon? Two of Britain’s best discuss</span>
+		</h1>
+		<div class="o-topper__standfirst">Henry Marsh and Stephen Westaby on ‘100 per cent mortality’, the embattled NHS and why self-deception is a clinical skill</div>
+	</div>
+
+	<div class="o-topper__background"></div>
+
+	<figure class="o-topper__visual">
+		<picture class="o-topper__picture">
+			<source media="(max-width: 490px)" srcset="https://www.ft.com/__origami/service/image/v2/images/raw/http://prod-upp-image-read.ft.com/194b5a92-9486-11e7-a9e6-11d2f0ebb7f0?source=next&amp;fit=scale-down&amp;quality=highest&amp;width=490 490w">
+			<source media="(max-width: 1220px)" srcset="https://www.ft.com/__origami/service/image/v2/images/raw/http://prod-upp-image-read.ft.com/37f4c8fc-9486-11e7-a9e6-11d2f0ebb7f0?source=next&amp;fit=scale-down&amp;quality=highest&amp;width=1220 1220w">
+			<source media="(min-width: 1220px)" srcset="https://www.ft.com/__origami/service/image/v2/images/raw/http://prod-upp-image-read.ft.com/13d2136c-9486-11e7-a9e6-11d2f0ebb7f0?source=next&amp;fit=scale-down&amp;quality=highest&amp;width=1440 1440w">
```

Comment:
Pedantic: 
- I think this final source is redundant as the `img` will be used if the max-width media queries don't match?

Question:
- Out of interest are we using `source`s over `srcset` for a reason? I'm on board with this 👍  to enforce high-res devices don't opt for a larger source despite being on a restricted internet connection, and to make it easier for implementers of `o-topper` to do art direction and change the image entirely for a smaller device.

Reply:
the demos are largely copypasta directly from Next article markup, so i don't really have any answers to your questions i'm afraid

Reply:
@notlee if I remember correctly the final source is the fallback for the picture tag. And also the `source` is the valid html for `picture` ? I may remember this incorrectly and @adgad may have a better memory for it.

Reply:
* `<picture>` over `<img srcset>` because it's art direction rather than optimisation (i.e. the small picture will be a different crop to the large one.
* So the middle one (`(max-width: 1220px)`) is the default so technically redundant. I feel like there was a reason we have as in the source, but don't remember now. It could just be that because we automatically generate the sources it was easier to keep it?